### PR TITLE
Add enable_mkldnn to static/deploy/python/infer.py

### PR DESCRIPTION
**Pull request by wozna**
This PR adds enable_mkldnn to `static/deploy/python/infer.py`. 
I'm adding this to make it easy to run static models with MklDNN.

### CODE REVIEW

**Comment by lidanqing-vv** [comment]
@lidanqing-intel

File: static/deploy/python/infer.py [hunk 1]
```diff
@@ -510,7 +521,7 @@ def predict_video(detector, camera_id):
     fps = 30
     width = int(capture.get(cv2.CAP_PROP_FRAME_WIDTH))
     height = int(capture.get(cv2.CAP_PROP_FRAME_HEIGHT))
-    fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+    fourcc = cv2.VideoWriter_fourcc(* 'mp4v')
```
Comment (yghstill) [inline static/deploy/python/infer.py]: Spaces here should be removed. (`* 'mp4v'` --> `*'mp4v'`)

Comment (wozna) [reply static/deploy/python/infer.py]: Oh, I can see that pre-commit fixed it like this, I'm gonna delete it.

**Review by nemonameless — APPROVED** [review]

**Review by nemonameless — COMMENTED** [review]
Please also modify in the develop branch.

**Comment by lidanqing-vv** [comment]
@wozna Please prepare same PR base on develop branch

**Review by yghstill — APPROVED** [review]

**Pull request merged**